_people/nick_sutton-smith.md

```diff
@@ -0,0 +1,10 @@
+---
+title: Nick Sutton-Smith
+headshot: Jx2LqS7
+gender: male
+course: English and History BA
```

Comment:
The Course needs to be on the line below

Reply:
Fixed